adapters/mobilefuse/mobilefuse.go

```diff
@@ -0,0 +1,193 @@
+package mobilefuse
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/golang/glog"
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/macros"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"net/http"
+	"strconv"
+	"text/template"
+)
+
+type MobilefuseAdapter struct {
```

Comment:
Nitpick: Is the proper capitalization Mobilefuse or MobileFuse? If it's the latter, consider renaming the adapter and similar variables.

Reply:
It is MobileFuse, but I thought it would be preferable to follow the same naming convention as the other bidders. eg. AppNexus, AdForm, AdKernel, CPMStar are Appnexus, Adform, Adkernel, Cpmstar. I can make the change, but I just want to make sure this is preferred first.

Reply:
I prefer it to make the branding clear, but that's just my opinion. If you'd like to leave it as is I'm fine.

Reply:
I've updated all relevant references in the latest commit.